tests/integration/test_storage.sh

```diff
@@ -69,20 +69,20 @@ trap teardown EXIT
 # If /etc/sysconfig/docker-storage-setup is missing, atomic should create the file.
 
 rm -f /etc/sysconfig/docker-storage-setup
-OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 2>&1)
+if [ -z "$VGROUP" ];then
+   VGROUP="test-atomic-storage"
+   OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 --vgroup $VGROUP 2>&1)
```

Comment:
> I am still fixing the tests

Why did you change the tests as part of this PR beyond just reverting #915?  Can we leave that out and worry about it in a separate PR?